apps/settings/elements/languages.html

```diff
@@ -29,6 +29,11 @@ <h1 data-l10n-id="language-header"> Language </h1>
             </select>
           </span>
         </li>
+        <li>
+          <a class="menu-item" id="more-languages" href="#">
```

Comment:
Sure, I'll use `class` here.  I'm curious though why the use of `id` is discouraged.  Can you shed some light on this guideline?